<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyWidget</class>
 <widget class="QWidget" name="ModifyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>姓名</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sexLabel">
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="sexLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="phoneLabel">
     <property name="text">
      <string>电话号码</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="phoneLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>房间号</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="idLineEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="modifyButton">
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="returnButton">
     <property name="text">
      <string>主菜单</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
